roles/epel/tasks/main.yml

```diff
@@ -5,7 +5,10 @@
 
 - name: Check if EPEL is not permanently enabled on RHEL
   fail: msg="Do not permanently enable EPEL on RHEL!"
-  when: epel_enabled and ansible_distribution == 'RedHat'
+  when:
+    - epel_enabled
+    - ansible_distribution == 'RedHat'
+    - qe_not_production_testing_server is not defined
```

Comment:
Shouldn't we define this variable for particular qe playbooks or qe roles in another commit of this pull request then?